2023-TK/0277 Rozendaal.txt

```diff
@@ -0,0 +1,4 @@
+https://www.rozendaal.nl/wp-content/uploads/2023/11/KM_C45823112315480-Proces-verbaal-TK2023.pdf
+https://www.rozendaal.nl/wp-content/uploads/2023/11/Controleprotocol.pdf
+https://www.rozendaal.nl/wp-content/uploads/2023/11/PV-Na-31-1-TK-2023.pdf
+
```

Comment:
Zou je deze laatste witregel nog kunnen verwijderen? Als ik deze zou pullen, dan zullen er uiteindelijk twee witregels aan het eind staan in het tekstbestand en daar struikelt download.sh dan over. Github de website toont volgens mij de laatste witregel niet, dus stiekem staan er nu twee ipv een. Of is dit ergens een instelling die ik verkeerd heb staan?

Reply:
Heb de witregel verwijdert. Bij mij staat er gewoon 1 witregel aan het eind in het tekstbestand. Nu werkt het in ieder geval sowieso voor je.